src/component/rocket/Rockets.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getRockets } from '../../Redux/rockets/rocketsSlice';
+import Rocket from "./Rocket"
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+
+  useEffect(() => {
+    if (rockets.length === 0) dispatch(getRockets());
+  }, [dispatch, rockets.length]);
+
+  return (
+    <div className="rocket-con">
+      {rockets
+        // eslint-disable-next-line max-len
```

Comment:
For the sake of maintaining standard best practices, kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. :wink: